.github/workflows/on-change.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: "Syncron"
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Any particular reason to go with ubuntu instead of a lighter weight distro like Alpine?

Reply:
GitHub only provides Windows, MacOs or Ubuntu. Anything else you have to host it yourself.

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners

Reply:
Can't we use a container with alpine like this? https://github.com/rhos-infra/znoyder/blob/main/.github/workflows/main.yml#L9

Reply:
I could spend some more time on that, but still, what would we want to achieve with it? Additionally, if I had to choose a distro I would go with CentOS or Fedora, which are the ones we are using for development.

Reply:
This is something work talking about on our sync later this week.
For now let's merge this.